dvc/repo/experiments/pull.py

```diff
@@ -1,9 +1,12 @@
 import logging
 
-from dvc.exceptions import DvcException, InvalidArgumentError
+from dvc.exceptions import DvcException
 from dvc.repo import locked
 from dvc.repo.scm_context import scm_context
+from dvc.scm import TqdmGit
+from dvc.types import IterableStr, Union
```

Comment:
`Union` should come from `typing` (unless we are actually overriding `typing.Union` in `dvc.types` it's better to be explicit here)